src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -417,30 +352,36 @@ def _backfill_assignments_for_co_admins(cli_ctx, auth_client, assignee=None):
                 assignee = _get_displayable_name(result[0]).lower()
             except ValueError:
                 pass
-        co_admins = [x for x in co_admins if assignee == _get_role_property(x, 'email_address').lower()]
+        co_admins = [x for x in co_admins if assignee == x.email_address.lower()]
 
     if not co_admins:
         return []
 
     result, users = [], []
     for i in range(0, len(co_admins), 10):  # graph allows up to 10 query filters, so split into chunks here
-        upn_queries = ["userPrincipalName eq '{}'".format(_get_role_property(x, 'email_address'))
+        upn_queries = ["userPrincipalName eq '{}'".format(x.email_address)
                        for x in co_admins[i:i + 10]]
         temp = list(list_users(graph_client.users, query_filter=' or '.join(upn_queries)))
         users += temp
     upns = {u.user_principal_name: u.object_id for u in users}
     for admin in co_admins:
         na_text = 'NA(classic admins)'
-        email = _get_role_property(admin, 'email_address')
+        email = admin.email_address
         result.append({
             'id': na_text,
             'name': na_text,
+        })
+        properties = {
             'principalId': upns.get(email),
             'principalName': email,
-            'roleDefinitionName': _get_role_property(admin, 'role'),
+            'roleDefinitionName': admin.role,
```

Comment:
this one also need to go thru worker?

Reply:
Same, classic-admin's api-version never changed.